src/builtins/BuiltinTemporal.cpp

```diff
@@ -591,6 +591,52 @@ static Value builtinTemporalDurationPrototypeAbs(ExecutionState& state, Value th
     return TemporalDuration::createTemporalDuration(state, std::abs(duration->getYear()), std::abs(duration->getMonth()), std::abs(duration->getWeek()), std::abs(duration->getDay()), std::abs(duration->getHour()), std::abs(duration->getMinute()), std::abs(duration->getSecond()), std::abs(duration->getMillisecond()), std::abs(duration->getMicrosecond()), std::abs(duration->getNanosecond()), nullptr);
 }
 
+static Value builtinTemporalPlainMonthDayConstructor(ExecutionState& state, Value thisValue, size_t argc, Value* argv, Optional<Object*> newTarget)
+{
+    if (!newTarget.hasValue()) {
+        ErrorObject::throwBuiltinError(state, ErrorObject::TypeError, ErrorObject::Messages::New_Target_Is_Undefined);
+    }
+
+    int referenceISOYear = 1972;
+
+    if (argc >= 4) {
+        if (argv[3].isInt32()) {
+            referenceISOYear = argv[3].asInt32();
+        } else {
+            ErrorObject::throwBuiltinError(state, ErrorObject::TypeError, "Invalid type");
+        }
+    }
+
+    return TemporalPlainMonthDay::createTemporalMonthDay(state, argv[0].asInt32(), argv[1].asInt32(), TemporalCalendar::toTemporalCalendarWithISODefault(state, argc >= 3 ? argv[2] : Value()).asObject()->asTemporalCalendarObject(), referenceISOYear);
+}
+
+static Value builtinTemporalPlainMonthDayFrom(ExecutionState& state, Value thisValue, size_t argc, Value* argv, Optional<Object*> newTarget)
+{
+    auto options = Temporal::getOptionsObject(state, argc >= 2 ? &argv[1] : nullptr);
```

Comment:
Please avoid allocating `nullptr` in `Value`. Value should have valid address only.
If you are about to present primitive types such as `null` or `undefined`, use `Value(Value::Null)` or `Value()` instead.

Reply:
check this again please.